### ISSUE TITLE
Source code for the visualizer

### ISSUE THREAD

**Issue by jackyko1991** [Crispher/MathlibExplorer#7]
Any chance that the source code of the visualizer will be released?

As mentioned in the intro video it is under C++ framework, I may help to modify it to add cross platform support.

**Comment by Crispher** [comment]
I might release the source after a major refactor. It'll be at least 2 month away from now.

**Comment by kciok** [comment]
@Crispher 
I wanna if there is a plan to open source? I am currently using Intel Mac and Linux, and hope to build it myself through the source code.

Thx!

**Comment by Crispher** [comment]
@Anonyame 
Yes. I am rewriting it in C# (Unity) and will release source code with the new version. Hopefully it will be done this month.
